README

```diff
@@ -353,6 +353,13 @@ WARNINGS and TAINT FLAGS:
         For more information on this issue consult
         https://www.freedesktop.org/wiki/Software/systemd/separate-usr-is-broken
 
+        systemd will warn if the filesystem is not usr-merged (ie: /bin, /sbin
+        and /lib* are symlinks to their counterparts under /usr). Taint flag
+        'unmerged-usr' will be set when this condition is detected.
```

Comment:
ie → i.e.

The sense of the phrase in parenthesis is reversed, a "not" is missing there.

Reply:
ops, missed the second one